.github/workflows/flow-semantic-release.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Semantic Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 20
+      - run: npm ci
+      - run: npm run build
+      - run: npx semantic-release
```

Comment:
Couple things here:
- Every step needs a name field
- Anywhere you're using an action you should be using a pinned commit.

```Yaml
# Current:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
           node-version: 20
      - run: npm ci
      - run: npm run build
      - run: npx semantic-release

# Becomes:
      - name: Checkout Code
        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
      - name: Configure Node JS
        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
        with:
           node-version: 20
      - name: Clean Install
        run: npm ci
      - name: Build Source
        run: npm run build
      - name: Run semantic-release
        run: npx semantic-release
```

Reply:
Hello @rbarkerSL,
Thanks for taking the time to review this PR. I have addressed you comment accordingly.